src/main/kotlin/org/treeWare/proto3/message/Proto3TypeConverter.kt

```diff
@@ -0,0 +1,47 @@
+package org.treeWare.proto3.message
+
+import org.treeWare.metaModel.FieldType
+import org.treeWare.metaModel.getFieldTypeMeta
+import org.treeWare.metaModel.getMetaName
+import org.treeWare.model.core.EntityModel
+import org.treeWare.model.core.getMetaModelResolved
+
+
+fun getEncodingType(fieldMeta: EntityModel): String =
```

Comment:
nit: when there is only 1 public function or class in a file, name the file after that function or class. Files use UpperCamelCase even if functions use lowerCamelCase, so this file should be named `GetEncodingType.kt`